cmake/packaging.cmake

```diff
@@ -184,8 +184,10 @@ set(CPACK_DEBIAN_${IRODS_PACKAGE_COMPONENT_POSTGRES_NAME_UPPERCASE}_PACKAGE_CONT
 set(CPACK_RPM_${IRODS_PACKAGE_COMPONENT_POSTGRES_NAME}_PACKAGE_NAME "irods-database-plugin-postgres")
 if (IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "opensuse")
   set(CPACK_RPM_${IRODS_PACKAGE_COMPONENT_POSTGRES_NAME}_PACKAGE_REQUIRES "${IRODS_PACKAGE_DEPENDENCIES_STRING}, irods-server = ${CPACK_RPM_PACKAGE_VERSION}-${CPACK_RPM_PACKAGE_RELEASE}, unixODBC, postgresql, psqlODBC")
-else()
+elseif (IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "centos" AND IRODS_LINUX_DISTRIBUTION_VERSION_MAJOR LESS 9)
```

Comment:
Why not `EQUALS 7`?

Just curious.

Reply:
This was working for 8, so wanted to box in the effect. I'm okay with either.

Reply:
If we can be explicit, let's do that. Just trying to guard against potentially hidden issues in the future.

Reply:
Gosh, I didn't even notice the bit about "centos". Yes, definitely this should be `EQUALS 7`.